tox.ini

```
@@ -53,9 +53,10 @@ commands =
     --out-dir {toxinidir}/dist/ \
     {toxinidir}
 
-[testenv:flake8]
-deps = flake8
-commands = flake8
+[testenv:lint]
```

I usually call it just
```suggestion
[testenv:pre-commit]
```

This also helps understand what's going on when you want to pass custom args via `tox -e pre-commit -- <custom_args_here>`.

---

Please lets not do this and I can explain why:
* tox -e lint is short and easy to type and future proof
* "pre-commit" is the current implementation of linting, maybe in the future we decide to sue something else, user/developer should not care about what to run to lint his change 

What we did on OpenStack which historically had lots of [testenv]s for each linter was to introduce `lint` or `linters` environment without removing the old ones and later (6mo-1y) removed them.  This was convenient for developers used to do just `tox -e flake9|pep8` as it have them time to adjust without swearing too often.

I am particularly opposed to use pre-commit because is more than double the number of chars to type, for something that any developer should run before creating a PR. Many forget to do it even when is short.

Does my reasoning makes sense?